.travis.yml

```diff
@@ -5,7 +5,15 @@ notifications:
 before_script:
   - npm prune
 after_success:
-  - npm run travis-deploy-once "npm run semantic-release"
+  - if [ ${TRAVIS_SECURE_ENV_VARS} = "true" ]; then
+      if [ "$TRAVIS_BRANCH" == "master" ]; then
+        npm run travis-deploy-once "npm run semantic-release";
+
+        git config --global user.name "Travis-CI";
+        git config --global user.email "travis@example.com";
+        npm run travis-deploy-once "npm run deploy";
+      fi
+    fi
```

Comment:
This doesnt quite feel right having the branch checks and `travis-deploy-once` 🤔

Reply:
Could just remove `travis-deploy-once` entirely since we're not running across multiple node versions.